### ISSUE TITLE
[Bug] UI is the wrong size in 1.13

### ISSUE THREAD

**Issue by dantman** [jrbudda/Vivecraft_113#9]
The UI was fine in 1.12 however it's the wrong size in 1.13. The inventory screen, crafting table/furnace screens, and the item bar are all half the size they were in 1.12. Additionally because of the wrong scale the collision areas you touch with your other hand to switch items does not line up with the items on the item bar. The far right item bar item collider is completely off the UI.

I do have a 4K TV. I'm not sure if this is relevant or not.

**Comment by ZwipZwapZapony** [comment]
"I do have a 4K TV. I'm not sure if this is relevant or not." I'm going to guess that it might be; If I have the GUI size in Vivecraft set to something other than automatic, then if I maximize the Vivecraft window, the GUI elements get smaller in-game. I'm guessing that the GUI is rendered to a dynamic texture at the size that it would in vanilla Minecraft, based on the window size, followed by Vivecraft then rendering that texture onto a fixed-size square, meaning that something filling 20% of a 100-pixel wide GUI will appear bigger in Vivecraft than something filling 10% of a 200-pixel wide GUI.

TL;DR: The GUI screens in Vivecraft seem to be of a fixed 3D size, regardless of how many pixels wide/tall the GUI screen's texture is, making 20 out of 100 pixels appear bigger on the 3D screen than 20 out of 200 pixels.
